diagnostic.md

```diff
@@ -178,4 +220,6 @@ with the class `.stretch` be at a _screen width_ of 550px?
 
 <!-- your answer starts here -->
 
+25px
+
```

Comment:
137.5 px - Reason being that since it's min-width, the media query will have .stretch be 25% and at 550px width, 25% is 137.5 px.